src/main/java/com/redhat/cloud/notifications/db/BehaviorGroupResources.java

```diff
@@ -0,0 +1,131 @@
+package com.redhat.cloud.notifications.db;
+
+import com.redhat.cloud.notifications.models.BehaviorGroup;
+import com.redhat.cloud.notifications.models.BehaviorGroupAction;
+import com.redhat.cloud.notifications.models.Bundle;
+import com.redhat.cloud.notifications.models.Endpoint;
+import com.redhat.cloud.notifications.models.EventType;
+import com.redhat.cloud.notifications.models.EventTypeBehavior;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import org.hibernate.reactive.mutiny.Mutiny;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.ws.rs.NotFoundException;
+import java.util.UUID;
+import java.util.logging.Logger;
+
+@ApplicationScoped
+public class BehaviorGroupResources {
+
+    private static final Logger LOGGER = Logger.getLogger(BehaviorGroupResources.class.getName());
+
+    @Inject
+    Mutiny.Session session;
+
+    public Uni<BehaviorGroup> createBehaviorGroup(BehaviorGroup behaviorGroup) {
+        return Uni.createFrom().item(behaviorGroup)
+                .onItem().transform(this::addBundleReference)
+                .onItem().transformToUni(session::persist)
+                .call(session::flush)
+                .replaceWith(behaviorGroup);
+    }
+
+    public Multi<BehaviorGroup> getBehaviorGroups(UUID bundleId, String accountId) {
+        String query = "FROM BehaviorGroup WHERE accountId = :accountId AND bundle.id = :bundleId";
+        return session.createQuery(query, BehaviorGroup.class)
+                .setParameter("accountId", accountId)
+                .setParameter("bundleId", bundleId)
+                .getResultList()
+                .onItem().transformToMulti(Multi.createFrom()::iterable);
+    }
+
+    public Uni<Boolean> updateBehaviorGroup(BehaviorGroup behaviorGroup) {
+        String query = "UPDATE BehaviorGroup SET name = :name, displayName = :displayName WHERE accountId = :accountId AND id = :id";
+        return session.createQuery(query)
+                .setParameter("name", behaviorGroup.getName())
+                .setParameter("displayName", behaviorGroup.getDisplayName())
+                .setParameter("accountId", behaviorGroup.getAccountId())
+                .setParameter("id", behaviorGroup.getId())
+                .executeUpdate()
+                .call(session::flush)
+                .onItem().transform(rowCount -> rowCount > 0);
+    }
```

Comment:
Not related to BehaviorGroup but to panache/hibernate. 
Do we need to do this manually? I though that we only had to get the `BehaviorGroup` object and set the values as we wanted for them to be updated.

Reply:
Yes and no 😃

The `BehaviorGroup` could be retrieved first (with a `SELECT` query or even a `session.find`) and then modified without having to write the `UPDATE` query in our code (changes would be commited at `flush` time). However, this would execute 2 SQL queries (`SELECT` first, `UPDATE` second) which is not ideal for performances. On the other hand, we can write the `UPDATE` query that way and get rid of the `SELECT` query. It's not always possible to do that though...

Hibernate sometimes has a bad reputation because it is said to be slow, but that's usually because there are multiple ways of doing the same thing and it's not always obvious which one is the best for performances.

Reply:
I don't think it would be that bad to do the `SELECT` first, specially to confirm that:
- User is part of the account (i.e. matches `accountId`)
- We aren't changing the `accountId`. We definitely don't want to change it ever.

Reply:
I'm not sure which version you reviewed since I pushed multiple times since your first comment, but the current code already checks the `accountId` and the update is restricted to the `name` and `displayName` fields. I really don't see how a `SELECT` would improve things here while it would certainly degrade performances.

Reply:
> I'm not sure which version you reviewed

Probably and old version.